### PR TITLE
chore: Add Loader component to App.tsx

### PR DESCRIPTION
**Pull request by marcosstevens2012**
### Before sending PR for review confirm that
- [ ] self review of PR is made and conventions are checked (check "Files changed" tab in PR) 
- [ ] all merge conflict are solved
- [ ] CICD checks passes

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -32,31 +31,7 @@ export function App(): React.ReactElement {
...
-                            }}
-                        >
-                            <p>{option.label}</p>
-                        </div>
-                    )}
-                    onChange={handleChange}
-                    onInputChange={handleInputValueChange}
-                />
-            </Card>
+            <Loader isLoading={false}> </Loader>
```
Comment (oscarmg11) [inline src/App.tsx:34]: It does not have an isLoading prop, no?

File: src/styles/loader.module.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #f0f0f0;
+  margin: 0;
+  flex-direction: column;
+}
+
+p {
```
Comment (oscarmg11) [inline src/styles/loader.module.css:12]: Don't query p elements in general, just change the p inside .loaderContainer, also the font family, why arial? we use popins use it, we already have a Text component, it is not useful here? Also the color, i'm pretty sure it should not be #333, cause it is not one of the defined colors we have

File: src/components/main.ts [hunk 3]
```diff
@@ -13,6 +12,8 @@ export * from 'src/components/Accordion';
 export * from 'src/components/DraggableList';
 export * from 'src/components/DraggableListItem';
 export * from 'src/components/DropDown';
+export * from 'src/components/Loader';
+export * from 'src/components/SlidingDots';
```
Comment (oscarmg11) [inline src/components/main.ts:16]: Do we need to expose the component SlidingDots to the consumer of the library? I don't think so, no? We just need the loader

File: src/components/Loader.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    return (
+        <div className={classes.loaderContainer}>
+            <SlidingDots />
+            <Text weight='semiBold'>Cargando, espera...</Text>
+        </div>
+    );
+}
+
+type Props = Omit<HTMLProps<HTMLSpanElement>, 'className'> & {
```
Comment (oscarmg11) [inline src/components/Loader.tsx:22]: Remove this props type, is not needed, cause you don't use the whole span attributes, you just use the loading prop, better use `LoaderProps` from the `Loader.d.ts` file

**Review by oscarmg11 — CHANGES_REQUESTED** [review]

File: src/components/main.ts (hunk 3 above)
Comment (marcosstevens2012) [reply src/components/main.ts:16]: the sliding dots replace the MIUI circular progress, you don't always need the loader, sometimes only the dots

File: src/components/SlidingDots.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+import classes from 'src/styles/slidingDots.module.css';
+
+export function SlidingDots(): React.ReactElement {
```
Comment (oscarmg11) [inline src/components/SlidingDots.tsx:7]: I think this component is not needed just put these div and span inside de `Loader` component

File: src/components/Loader.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+import { SlidingDots } from 'src/components/SlidingDots';
+import { Text } from 'src/components/Text';
+import classes from 'src/styles/loader.module.css';
+import { LoaderProps } from 'src/types/components/Loader';
+
+export function Loader({ loading, loadingText }: LoaderProps): React.ReactElement | null {
```
Comment (oscarmg11) [inline src/components/Loader.tsx:10]: Just `text` instead of `loadingText` is better

**Review by oscarmg11 — CHANGES_REQUESTED** [review]

**Pull request merged**